src/providers/CogsConnectionProvider.tsx

```diff
@@ -74,14 +82,30 @@ export default function CogsConnectionProvider({
     };
   }, [hostname, port]);
 
+  const audioPlayerRef = useRef<CogsAudioPlayer>();
+  useEffect(() => {
+    if (audioPlayer && !audioPlayerRef.current && connectionRef.current) {
```

Comment:
Should this technically null out `audioPlayerRef.current` when `audioPlayer` is `false`? Or does that not really matter? (Same question for video below)

Reply:
Yes, technically it should. If we want to support this it should also:
- Stop playing any audio/video clips
- Remove those audio/video clip from Howler/DOM
- Remove all listeners on `CogsConnection`

At that point the `AudioPlayer`/`VideoPlayer` instance and anything they created should get garbage collected. 

The alternative is we assume you will not change the value of `audioPlayer` and `videoPlayer`. I think this is the common case. To make it clear we could add some to typedoc for `CogsConnectionProvider` warning you that if you later set the parameter to `false` it will be ignored.

What do you reckon?

Reply:
Yeah seems overkill to support an edge case we can't even foresee being ever used. Happy just to document as you say.

Reply:
Done.